CatchMe/CatchMe/Source/ViewControllers/SettingVC.swift

```diff
@@ -0,0 +1,117 @@
+
+    //MARK: - Life Cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupLayout()
+        configUI()
+        setTableView()
+    }
+
+    //MARK: - Custom Method
+    func setupLayout() {
+        view.addSubviews([backButton, titleLabel, settingTableView])
+        
+        backButton.snp.makeConstraints { make in
+            make.top.equalTo(view.snp.top).offset(55)
+            make.leading.equalTo(view.snp.leading).offset(14)
+            make.width.height.equalTo(48)
+        }
+        
+        titleLabel.snp.makeConstraints { make in
+            make.top.equalTo(view.snp.top).offset(66)
+            make.centerX.equalTo(view.snp.centerX)
+            make.centerY.equalTo(backButton.snp.centerY)
+        }
+        
+        settingTableView.snp.makeConstraints { make in
+            make.top.equalTo(titleLabel.snp.bottom).offset(40)
+            make.leading.equalToSuperview().offset(28)
+            make.trailing.equalToSuperview().inset(28)
+            make.bottom.equalToSuperview()
+        }
+    }
+    
+    func configUI() {
+        view.backgroundColor = .black100
+        
+        backButton.backgroundColor = .blue100
+        
+        titleLabel.text = "설정"
+        titleLabel.textColor = .white
+        titleLabel.font = UIFont.stringBoldSystemFont(ofSize: 20)
+    }
+    
+    func setTableView() {
+        settingTableView.dataSource = self
+        settingTableView.delegate = self
+        
+        settingTableView.register(SettingTVC.self, forCellReuseIdentifier: "SettingTVC")
+        
+        settingTableView.backgroundColor = .clear
+        settingTableView.separatorStyle = .none
+        settingTableView.tableFooterView = UIView(frame: .zero)
+        settingTableView.sectionFooterHeight = 0
+    }
+}
+
+// MARK: - UITableViewDelegate
+extension SettingVC: UITableViewDelegate {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return sectionList.count
+    }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: false)
+    }
+    
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 54
+    }
+    
+    func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+        return 16
+    }
+    
+    func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        return UIView()
+    }
+}
+
+// MARK: - UITableViewDataSource
+extension SettingVC: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "SettingTVC", for: indexPath)
```

Comment:
```suggestion
        guard let cell = tableView.dequeueReusableCell(withIdentifier: "SettingTVC", for: indexPath) as? SettingTVC else { return UITableViewCell() }
```
cell를 그냥 가져와도 상관없지만 혹시나 nil로 cell이 들어온다면 앱 자체가 터져버릴 수도 있기 때문에 항상 guard let으로 cell를 가져와주는게 좋아요.!